multinet/auth/types.py

```diff
@@ -0,0 +1,54 @@
+"""Types associated with authentication."""
+
+from typing_extensions import TypedDict
+
+
+class GoogleUserInfo(TypedDict):
+    """Representation of raw User Info from a Google Account."""
+
+    family_name: str
+    given_name: str
+    name: str
+    picture: str
+    locale: str
+    sub: str  # Unique identifier
+    email: str
+    email_verified: bool
+
+    # Time of token expiration in Unix Seconds
+    exp: int
+
+    # Time of token issue
+    iat: int
+
+    # Issuer
+    iss: str
+
+    aud: str
+    azp: str
+    at_hash: str
+    hd: str
```

Comment:
I'd love a definition for these pieces, please. Also if they're not absolutely necessary, should we track them?

Reply:
They're not necessary for the data that we store, but it does represent the response that we receive from Google, so I think it's worth keeping. The main reason I didn't explain most of those is because we don't actually use them, so I didn't think it necessary, but if we think it'll provide clarity I can add comments.

Reply:
Added via d8b455f